HousingRegisterApi/V1/Controllers/ApplicationsApiController.cs

```diff
@@ -137,6 +139,17 @@ public IActionResult CompleteApplication([FromRoute][Required] Guid id)
             return Ok(result);
         }
 
+        [ProducesResponseType(typeof(ApplicationResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPost]
+        [Route("bedrooms")]
+        public IActionResult CalculateBedrooms([FromBody] CalculateBedroomsRequest request)
```

Comment:
I was expecting to see an `{id}` here since the API should be functional by itself, i.e. retrieve the application and return the number of bedrooms. This relies on already populated data from elsewhere.

Reply:
Ah yes, aslong as the application and applicants exists in the database prior to this call.. i can fix that

Reply:
fixed